rooms.js

```diff
@@ -261,6 +261,22 @@ class Room {
 			}
 		}
 	}
+
+	get metadata() {
+		let settings = {
+			// settings
+			privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'),
+			modchat: this.modchat,
+			modjoin: this.modjoin,
+
+			// games
```

Comment:
Isnt this just an extra ~150 characters

``|metadata|{"title":"Lobby","id":"lobby","privacy":"public","type":"chat","modchat":false,"modjoin":false,"auth":{},"uno":true,"tournaments":"#","hangman":true}``